bigbone/src/main/kotlin/social/bigbone/MastodonClient.kt

```diff
@@ -284,6 +285,27 @@ private constructor(
     @get:JvmName("timelines")
     val timelines: TimelineMethods by lazy { TimelineMethods(this) }
 
+    /**
+     * Access API methods under "preferences" endpoint.
+     */
+    @Suppress("unused") // public API
+    @get:JvmName("preferences")
+    val preferences: PreferenceMethods by lazy { PreferenceMethods(this) }
```

Comment:
Could you please bring your methods added into alphabetical order?

Reply:
Sure!